src/app/reducers/environments.ts

```diff
@@ -17,16 +18,27 @@ export interface State {
 }
 
 export const getInitialEnvironmentState = (): EnvironmentState => {
+  const { initialData } = getAltairConfig();
   return {
     title: 'Environment',
-    variablesJson: '{}'
+    variablesJson: '{}',
+    id: uuid(),
+    ...initialData.environments.base
```

Comment:
We should make this more explicit, rather than extending the object with what the user provides. It makes it easier to track down bugs in the future, especially since this is a user provided input.

What I mean is doing something like `title: initialData.environments.base.title` and so on.

Reply:
The reason I did it like this is if the user happened to not define `environments.base` then setting the title explicitly like that would throw. The other option is to null-guard along the way, which is kinda ugly haha:

```
title: (initialData.environments.base && initialData.environments.base.title) || 'Environment'
```

Reply:
No I only gave that example to explain what I mean by "explicit". It should ideally be handled in a more elegant way, like first checking that the `environments.base` exist before using it